modules/seqexec/server/src/main/scala/seqexec/server/StepType.scala

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.server
+
+import seqexec.model.enum.Instrument
+
+sealed trait StepType {
+  val instrument: Instrument
+  val includesObserve: Boolean = true
```

Comment:
Now step may skip the observation if e.g. they are align and calib which doesn't require one